src/lava/proc/rf/process.py

```diff
@@ -68,8 +68,8 @@ def __init__(self,
         if state_exp > 0:
             vth = int(vth * (1 << state_exp))
```

Comment:
Should this cast be also np.int32() instead of int()?

Reply:
From my understanding this does not need to be casted to np.in32 because all the neurons should share the same voltage threshold. However, if you are supporting (or one day will support) the functionality of having a separate voltage threshold for each neuron then we should this into a np.32() cast also.